src/settings-ui/Settings.UI/Views/AwakePage.xaml

```diff
@@ -73,6 +73,12 @@
                         </controls:Setting.ActionContent>
                     </controls:Setting>
 
+                    <controls:Setting x:Uid="Awake_EnableAudioKeepAwake" Icon="&#xE995;">
```

Comment:
E995 = "Volume3"
E767 = "Volume"

Reply:
True, visually identical, just different intention. Same like "chromeclose" and "calculatormultiply": both are `X`.

Reply:
![image](https://user-images.githubusercontent.com/28542943/156028656-4978fbca-813e-41ea-9eef-a9d0ffe43824.png)

Updated!

Reply:
It would be so much easier if we could actually use those names in stead of the numbers...